artdaq-core-mu2e/Overlays/DTC_Packets.cpp

```diff
@@ -952,6 +959,12 @@ void DTCLib::DTC_SubEvent::SetupSubEvent()
 	auto ptr = reinterpret_cast<const uint8_t*>(buffer_ptr_);
 
 	memcpy(&header_, ptr, sizeof(header_));
+	if(header_.subevent_format_version != REQUIRED_SUBEVENT_FORMAT_VERSION)
+	{
+		TLOG(TLVL_ERROR) << "A DTC_WrongPacketTypeException occurred while setting up a DTC Subevent in the header format version 0x" <<
```

Comment:
I'll note that this code will break all existing data files (e.g. GR-1), since it will no longer be able to read the raw data from DTC_SubEvents